client/landing/stepper/declarative-flow/internals/steps-repository/subscribers/index.tsx

```diff
@@ -1,12 +1,15 @@
 import { StepContainer } from '@automattic/onboarding';
 import { ReactElement } from 'react';
 import { useSetupOnboardingSite } from 'calypso/landing/stepper/hooks/use-setup-onboarding-site';
+import { useSite } from 'calypso/landing/stepper/hooks/use-site';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import type { Step } from '../../types';
 
 const Subscribers: Step = function ( { navigation } ): ReactElement | null {
 	const { submit } = navigation;
-	useSetupOnboardingSite();
+	const site = useSite();
+
+	useSetupOnboardingSite( { site } );
```

Comment:
Could be helpful to put this inside a useEffect to only run it once?

Reply:
It's not needed since the memo has the `siteIsLoaded` dependency which will ensure it is executed only once
https://github.com/Automattic/wp-calypso/pull/66709/files#diff-4d505341296564cfffbd86a21a2c5694e77b1e92c7c2db49b4cea374e1b62ee7L70

Reply:
Haha oops I missed that. 👍